client/landing/stepper/declarative-flow/flows/onboarding/onboarding.ts

```diff
@@ -401,56 +396,7 @@ const onboarding: Flow = {
 			}
 		};
 
-		const goBack = () => {
-			switch ( currentStepSlug ) {
-				case 'use-my-domain':
-					if ( getQueryArg( window.location.href, 'step' ) === 'transfer-or-connect' ) {
-						const destination = addQueryArgs( '/use-my-domain', {
-							...getQueryArgs( window.location.href ),
-							step: 'domain-input',
-							initialQuery: getQueryArg( window.location.href, 'initialQuery' ),
-						} );
-						return navigate( destination );
-					}
-
-					if ( window.location.search ) {
-						window.history.replaceState( {}, document.title, window.location.pathname );
-					}
-					return navigate( 'domains' );
-				case 'plans':
-					if ( redirectedToUseMyDomain ) {
-						if ( Object.keys( useMyDomainQueryParams ).length ) {
-							// restore query params
-							const useMyDomainURL = addQueryArgs( 'use-my-domain', useMyDomainQueryParams );
-							return navigate( useMyDomainURL );
-						}
-						return navigate( 'use-my-domain' );
-					}
-					return navigate( 'domains' );
-				case 'domains':
-					if ( isGoalsAtFrontExperiment ) {
-						if ( isBigSkyBeforePlansExperiment && createWithBigSky ) {
-							return navigate( 'design-choices' );
-						}
-						return navigate( 'design-setup' );
-					}
-				case 'design-setup':
-					if ( isDesignChoicesStepEnabled ) {
-						return navigate( 'design-choices' );
-					}
-					if ( isGoalsAtFrontExperiment ) {
-						return navigate( 'goals' );
-					}
-				case 'difmStartingPoint':
-					return navigate( 'goals' );
-				case 'design-choices':
-					return navigate( 'goals' );
-				default:
-					return;
-			}
-		};
```

Comment:
This is the kind of simplification we love to see! Thanks for seeing it through 🌟